README.md

```diff
@@ -263,12 +265,21 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller
 
 We have provided a sample node.json to add a node in kubernetes. Please make sure edge-node is added in kubernetes. Run below steps to add edge-node.
 
-```shell
-kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json
-```
-
-Run Edge
-
++ Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/build/node.json` file and change `metadata.name` to the IP of the edge node
++ Deploy node
+    ```shell
+    kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/node.json
+    ```
+
+Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/edge/conf/edge.yaml` configuration file
++ Replace `edgehub.websocket.certfile` and `edgehub.websocket.keyfile` with your own certification path
++ In websocket:url you need to give IP address of the master.
++ replace `fb4ebb70-2783-42b8-b3ef-63e2fd6d242e` with edge node ip in edge.yaml in 3 place.
```

Comment:
+ replace `fb4ebb70-2783-42b8-b3ef-63e2fd6d242e` with edge node ip in edge.yaml for the below fields : 
    + `websocket.url`
    + `controller.node-id`
    + `edged:hostname-override`